aea/mail/base.py

```diff
@@ -252,6 +252,7 @@ def in_queue(self) -> queue.Queue:
     @property
     def out_queue(self) -> asyncio.Queue:
         """Get the out queue."""
+        assert self._out_queue is not None, "Accessing out queue before loop is started."
```

Comment:
this is needed because we cannot set the queue in the constructor.

Reply:
OK